### ISSUE TITLE
synthesize.sh script's argument -s does not work anymore

### ISSUE THREAD

**Issue by vijay-murali** [capergroup/bayou#85]
```
./synthesize.sh --num_samples 200 Test.java 
[2017-10-10 13:42:21,611] ERROR in app: Exception on / [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.5/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/vm17/Work/bayou/tool_files/build_scripts/out/python/bayou/server/ast_server.py", line 170, in <lambda>
    http_server.add_url_rule("/", "index", lambda: _handle_http_request(bp), methods=['POST'])
  File "/home/vm17/Work/bayou/tool_files/build_scripts/out/python/bayou/server/ast_server.py", line 37, in _handle_http_request
    asts = _handle_generate_asts_request(request_dict, predictor)
  File "/home/vm17/Work/bayou/tool_files/build_scripts/out/python/bayou/server/ast_server.py", line 53, in _handle_generate_asts_request
    asts = _generate_asts(evidence_json_str, predictor, num_samples=sample_count, max_ast_count=max_ast_count)
  File "/home/vm17/Work/bayou/tool_files/build_scripts/out/python/bayou/server/ast_server.py", line 64, in _generate_asts
    if num_samples < 1:
TypeError: unorderable types: str() < int()
```

**Comment by rmbarnett-rice** [comment]
Can reproduce. Acceptance test created.

**Comment by rmbarnett-rice** [comment]
accidentally did these changes on issue-91 branch, but awaiting merge

**Comment by vijay-murali** [comment]
Merged PR #94.
